examples/browser-webrtc/server/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+authors = ["Doug Anderson <douganderson444@peerpiper.io>"]
+edition = "2021"
+license = "MIT"
+name = "webrtc-example-server"
+publish = false
+rust-version = { workspace = true }
+version = "0.1.0"
+
+[[bin]]
+name = "webrtc-example-server-bin"
+path = "src/bin/webrtc-example-server.rs"
```

Comment:
I'd prefer convention over configuration and rename the file to `main.rs`.

Reply:
Well, I did just change this from `main.rs` to address @mxinden's previous note, so to avoid further back and forth perhaps we should all agree what this should be named before I rename it again.